meta-resin-common/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -24,6 +24,10 @@ EXTRA_OECONF += " \
     "
 PACKAGECONFIG_append = " modemmanager ppp"
 
+# The external DHCP client doesn't work well with our`ipv4.dhcp-timeout`
+# configuration. Switch to the internal one.
+PACKAGECONFIG_remove = "dhclient"
+
```

Comment:
NIT: Should be `SIGFPE` instead of `SIGFPES` int he commit log

Reply:
Fixed typo.

Reply:
This networkmanager dhcp change diff appears in the modemmanager move resin specific files commit?

Reply:
Fixed. Good catch.